src/cdk/table/row.ts

```diff
@@ -294,3 +294,11 @@ export class CdkFooterRow {
 })
 export class CdkRow {
 }
+
+/** Row that can be used to display a message when no data is shown in the table. */
+@Directive({
+  selector: 'ng-template[cdkEmptyPlaceholderRowDef]'
+})
+export class CdkEmptyPlaceholderRowDef {
+  constructor(public templateRef: TemplateRef<any>) {}
```

Comment:
```suggestion
  constructor(public templateRef: TemplateRef<unknown>) {}
```
I just realized we should probably start making these `unknown` instead of `any`